Add `Tournament Committee applications now open!` news post (#12659)
ppy/osu-wiki: news/2025/2025-01-12-tournament-committee-applications-now-open.md

---
layout: post
title: Tournament Committee Applications Now Open!
date: 2025-01-12 15:00:00 +0000
---

Do you want to help shape the future of the osu! tournament community? If so, then apply now to join the [Tournament Committee (TC)](/wiki/People/Tournament_Committee) during our first ever open application cycle!

> 

Reviewer: ```suggestion

![](/wiki/shared/news/2025-01-12-tournament-committee-applications-now-open/banner.jpg)

```

We are seeking passionate individuals with significant tournament experience—including players—to help support and improve the community.

## What Does the Tournament Committee Do?

The TC plays a vital role in maintaining and supporting the growth of the tournament scene. As a member, your primary responsibilites will include:

> - **Tournament Reviews**: Approving or denying badge support for tournaments and communicating any necessary changes to organisers.

Reviewer: we put the ":" inside the bolded text in similar lists

Author: Fixed

- **Handling Reports and Inquiries**: Investigating tournament-related issues reported by the community, assessing the validity of claims, and providing timely and well-considered responses to users. 
- **Maintaining the [Official tournament support](/wiki/Tournaments/Official_support) wiki**: Keeping official rules and documentation up to date.

> - **Mentoring the Community**: Guiding newer members of the tournament scene towards making better decisions.

Author: This is the best I could come up with, not sure if it adds much or works better than the existing language.

```
- **Mentoring the community:** Guiding newer members of the tournament scene towards making better decisions, providing resources, and answering tournament-related questions.
- ```

- **Community Leadership**: Acting as a representative and leader within the tournament space.

## Requirements for joining

To join the TC, you must meet the following minimum criteria:

- **Age**: You must be at least 18 years of age.
- **Account Standing**: All applicants will undergo a security check. Any notable disciplinary actions within the last 12 months will disqualify a candidate.
- **Language Proficiency**: Fluency in English is mandatory, as most reviewed content and all communications are in English.

## What We Look for in Candidates

In addition to meeting the requirements, successful candidates typically demonstrate the following qualities:

> - **Extensive Tournament Experience**: Has **2 years** of recent, meaningful involvement in tournaments as a player, staff member, or organiser. Community members will approach you with complex issues and expect you to provide sound guidance.
> - **Maturity and Professionalism**: Communicates respectfully and handles responsibilities with professionalism.
> - **Composure and Critical Thinking**: Displays a level-headed approach to analyzing multi-faceted situations while providing reasonable insight.
> - **Good Behaviour**: Possesses a **well-established reputation** within the community and tournaments.

Reviewer: ```suggestion
- **Extensive tournament experience**: **2 years** of recent, meaningful involvement in tournaments as a player, staff member, or organiser. Community members will approach you with complex issues and expect you to provide sound guidance.
- **Maturity and professionalism**: Ability to communicate respectfully and handle responsibilities with professionalism.
- **Composure and critical thinking**: A level-headed approach to analysing multi-faceted situations while providing reasonable insight.
- **Good behaviour**: A **well-established reputation** within the community and tournaments.
```

> ## Standout Qualities (Bonus Points!)

Reviewer: ```suggestion
## Standout qualities (bonus points!)
```

We are particularly interested in candidates who:

- Have a clear understanding of the tournament hosting process and badging criteria.
- For staff members: Have hosted multiple badged tournaments or have experience as a tournament player.
- For players: Have staffed multiple tournaments or hosted at least one badged tournament.
- Consistently demonstrate a genuine desire to improve the tournament scene through their actions and advocacy.
- Are fluent in languages other than English.

> # Decision Process

Reviewer: ```suggestion
# Decision process
```

All individuals under consideration will first be brought to open discussion, followed by an anonymous voting period. Following this vote, we will notify each candidate of the decision via email.

> For those selected, we will invite you to an optional 20-minute call with two randomly assigned TC members. This call serves as an opportunity to ask any questions you may have about the team, workload, or anything else before deciding to accept this role. We want you to feel confident and fully informed before making your decision to join the team. 

Reviewer: so a job interview lol
certainly haven't heard any other osu! team do this

Author: This is specifically not labeled as an interview, it's optional and meant to be informative for anyone considering joining (who was selected). The TC workload is quite large.

*Please note that these calls are informal and focused on open conversation. There's no need to prepare, and we're happy to provide insights into our team and process.*

## Application

> **[Click here to apply!](https://forms.gle/uADXw7zHM81e584y8)** After authentication, you'll complete a Google Form that collects basic information and includes scenario-based questions. We recommend setting aside about 30 minutes to provide well-thought-out and detailed answers. To ensure a fair evaluation process, please respond as if we're unfamiliar with you or your previous work, even if you're well known in the community. This approach helps us gain a clear and comprehensive understanding of your skills, perspective, and enthusiasm.

Reviewer: ```suggestion
**[Click here to apply!](https://forms.gle/uADXw7zHM81e584y8)**

After authentication, you'll complete a Google Form that collects basic information and includes scenario-based questions. We recommend setting aside about 30 minutes to provide well-thought-out and detailed answers.

To ensure a fair evaluation process, please respond as if we're unfamiliar with you or your previous work, even if you're well known in the community. This approach helps us gain a clear and comprehensive understanding of your skills, perspective, and enthusiasm.
```

> **[Click here to apply!](https://forms.gle/uADXw7zHM81e584y8)** After authentication, you'll complete a Google Form that collects basic information and includes scenario-based questions. We recommend setting aside about 30 minutes to provide well-thought-out and detailed answers. To ensure a fair evaluation process, please respond as if we're unfamiliar with you or your previous work, even if you're well-known in the community. This approach helps us gain a clear and comprehensive understanding of your skills, perspective, and enthusiasm.

Reviewer: doesn't modify a noun

```suggestion
**[Click here to apply!](https://forms.gle/uADXw7zHM81e584y8)** After authentication, you'll complete a Google Form that collects basic information and includes scenario-based questions. We recommend setting aside about 30 minutes to provide well-thought-out and detailed answers. To ensure a fair evaluation process, please respond as if we're unfamiliar with you or your previous work, even if you're well known in the community. This approach helps us gain a clear and comprehensive understanding of your skills, perspective, and enthusiasm.
```

## Timeline

This application cycle will remain open for **2 weeks**, closing on **Sunday, January 26, 2025**. Please note that it may take up to 6 weeks after the closing date to process all applications, so we appreciate your patience.

## Questions

If you have any questions about the process, feel free to:

- Leave a comment on this news post.
- Contact us via [tournaments@ppy.sh](mailto:tournaments@ppy.sh).

> - Join the [osu! Discord server](https://discord.gg/ppy) and visit the `#tournaments` channel to chat with us.

Reviewer: skips the redirect, thus decreasing load time

```suggestion
- Join the [osu! Discord server](https://discord.com/invite/ppy) and visit the `#tournaments` channel to chat with us.
```

> Thank you for taking the time to apply and share your passion with us—your efforts make all the difference in shaping the future of our community!

Reviewer: ```suggestion
Thank you for taking the time to apply and share your passion with us — your efforts make all the difference in shaping the future of our community!
```

> -ChillierPear & Stage

Author: I'm confused, we looked at similar news posts and the authors were credited rather than the team.

> -ChillierPear & Stage

Author: I see now that the second suggestion was just something to consider, sorry. Didn't see the first suggestion.
